src/routes/safe/components/Transactions/TxList/InfoDetails.tsx

```diff
@@ -1,16 +1,26 @@
-import { Text } from '@gnosis.pm/safe-react-components'
 import { ReactElement, ReactNode } from 'react'
+import { Text } from '@gnosis.pm/safe-react-components'
+
+import { StyledDetailsTitle, StyledTxInfoDetails } from 'src/routes/safe/components/Transactions/TxList/styled'
 
 type InfoDetailsProps = {
   children: ReactNode
-  title: string
+  title: string | ReactElement
+  uppercase?: boolean
 }
 
-export const InfoDetails = ({ children, title }: InfoDetailsProps): ReactElement => (
-  <>
-    <Text size="xl" strong>
-      {title}
-    </Text>
+export const InfoDetails = ({ children, title, uppercase = true }: InfoDetailsProps): ReactElement => (
+  <StyledTxInfoDetails>
+    {uppercase ? (
+      <StyledDetailsTitle size="sm" strong color="placeHolder">
+        {title}
+      </StyledDetailsTitle>
+    ) : (
+      <Text size="sm" strong color="placeHolder">
+        {title}
+      </Text>
+    )}
```

Comment:
That's not [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
Can't uppercase just be a CSS prop on the text?

Reply:
Totally! Thanks for the idea